CHANGES.md

```diff
@@ -3,7 +3,7 @@ Release Changes List
 ====================
 
 ==============
-version 1.13.0
+version 1.13.1 (PRE-RELEASE)
```

Comment:
Makes sense. This was just a placeholder anyhow, like "to be determined". Whichever branch you put your changes in, we can just pick it to the other one.